Add Policy Packs for Azure from ext sheet. (#777)
turbot/guardrails-samples: policy_packs/azure/storage/enforce_creator_timestamp_tags_for_storage_accounts/README.md

---
categories: ["tagging"]
---

# Enforce Creator and Create Timestamp Tags for Azure Storage Account

Enforcing Creator and Create Timestamp tags for Azure Storage Accounts ensures that each storage account is tagged with information about who created it and when it was created. This practice enhances resource management, accountability, and auditing by providing clear metadata for tracking and monitoring purposes.

This [policy pack](https://turbot.com/guardrails/docs/concepts/resources/smart-folders) can help you configure the following settings for storage accounts:

> - Enforce creator and create timestamp tags for storage accounts

Reviewer: ```suggestion
- Add creator and createTimestamp tags on the storage account
```
